mir-ci/mir_ci/tests/robot/platforms/wayland/WaylandHid.py

```diff
@@ -94,6 +98,15 @@ async def release_pointer_buttons(self) -> None:
         for button in Button:
             self.button(button, False)
 
+    @keyword
+    async def vertical_scroll(self, amount: float) -> None:
+        await self.connect()
+        if self.pointer:
+            self.pointer.axis_source(0)
+            self.pointer.axis(self.timestamp(), 0, amount)
+            self.pointer.frame()
+        self.display.roundtrip()
```

Comment:
I think this is excessive, we should be able to do without scrolling (just selecting buttons that are on screen at all times).